Qualisys/QTM-Unity-Realtime-Streaming/Streaming/RTSkeleton.cs

```diff
@@ -107,9 +101,9 @@ private void BuildMecanimAvatarFromQtmTPose()
             {
                 name = this.SkeletonName,
                 position = Vector3.zero,
-                rotation = Quaternion.identity,
+                rotation = Quaternion.AngleAxis(180, Vector3.up), //TODO: Is it possible to calculate this?
```

Comment:
"`TODO`"

Reply:
In QTM default poses are facing +Y which becomes -Z in Unity.
We rotate 180 degrees to match unity forward.